lld/test/ELF/gnu-nocrossrefs.s

```diff
@@ -0,0 +1,150 @@
+// REQUIRES: x86
+// UNSUPPORTED: system-windows
```

Comment:
Why is windows prohibited? The test doesn't seem to use backslashes.

A better place is `linkerscript/nocrossrefs.test` using `rm -rf %t && split-file %s %t && cd %t`

Reply:
For some reason this test failed on windows with smth like `unsupported file type gnu-crossrefs.o` during link stage. 

 Maybe you have an idea why that happened?

Reply:
Rewrote test and dropped windows part. Let's see if it will work